src/main/features/linux/mprisService.js

```diff
@@ -0,0 +1,84 @@
+import { app } from 'electron';
+import mpris from 'mpris-service';
+import _ from 'underscore';
+
+function mprisService() {
+  const mainWindow = WindowManager.getAll('main')[0];
+  let _songInfo = {};
+
+  const player = mpris({
+    name: 'google-play-music-desktop-player',
+    identity: 'Google Play Music Desktop Player',
+    canRaise: true,
+    supportedInterfaces: ['player'],
+    desktopEntry: 'google-play-music-desktop-player',
+  });
+  player.playbackStatus = 'Stopped';
+
+  player.on('raise', () => {
+    mainWindow.setSkipTaskbar(false);
+    mainWindow.show();
+  });
+
+  player.on('quit', () => {
+    app.quit();
```

Comment:
This quit should be received only if the mpris client sends a quit, generally speaking the client might want to switch to a different media source, so quitting should in fact quit. The spec does say that this should cause the client to close, but doesn't have to be implemented.